Documentation/Usage.md

```diff
@@ -397,3 +398,34 @@ manifest API).
 It is *not* expected the packages would ever use this feature unless absolutely
 necessary to support existing clients. In particular, packages *should not*
 adopt this syntax for tagging versions supporting the _latest GM_ Swift version.
+
+### Editable Packages
+
+SwiftPM supports editing dependencies, which means moving it into a location under `Packages/` directory. 
```

Comment:
I think this should say more about the *why* rather than that it moves it into Packages. For example, "support editing dependencies, when your work requires making a change to one of your dependencies (for example, to fix a bug, or add a new API)."